frontend/src/components/LoginSignupModal/LoginSignupModal.jsx

```diff
@@ -0,0 +1,143 @@
+
+    // Step 1: Sign Up
+    let signUpResponse = await fetch(`${baseUrl}/api/auth/signup`, {
+        method: 'POST',
+        headers: {
+        'Content-Type': 'application/json',
+    },
+        body: JSON.stringify({ firstName: firstName, lastName: lastName, email: signUpEmail, password: signUpPassword, username: username })
+    });
+
+    // Step 2: Log In
+    if (signUpResponse.status === 201) {
+    // GOOD NEWS.
+    setsignupStatus("You have signed up succesfully, please login")
+    } else {
+    setErrorMessage("An error has occured, please try again")
+    }
+}
+
+// For login
+const handleEmailChange = (event) => {
+    setEmail(event.target.value)
+}
+
+const handlePasswordChange = (event) => {
+    setPassword(event.target.value)
+}
+
+// For Signup
+const handleSignupEmailChange = (event) => {
+    setSignupEmail(event.target.value)
+}
+
+const handleSignupPasswordChange = (event) => {
+    setSignupPassword(event.target.value)
+}
+
+const handleFirstNameChange = (event) => {
+    setfirstName(event.target.value)
+}
+
+const handleLastNameChange = (event) => {
+    setlastName(event.target.value)
+}
+
+const handleUserNameChange = (event) => {
+    setuserName(event.target.value)
+}
+
+    return (
+    <>
+        <div className="form-container-sign-up-container">
+            <form onSubmit={handleSignUpSubmit}>
+            <h1 >Create Account</h1>
+            <input id="first-name" type="text" placeholder="First Name" value={ firstName } onChange={handleFirstNameChange} />
+            <input id="last-name" type="text" placeholder="Last Name" value={ lastName } onChange={handleLastNameChange} />
+            <input id="user-name" type="text" placeholder="Username" value={ username } onChange={handleUserNameChange} />
+            <input id="sign-up-email" type="email" placeholder="Email" value={ signUpEmail } onChange={handleSignupEmailChange} />
+            <input id="sign-up-password" type="password" placeholder="Password" value={ signUpPassword } onChange={handleSignupPasswordChange} />
+            {signupStatus && (
+            <div className="error-message">
+
+                <p >{signupStatus}</p>
+            </div>
+            )}
+            <button id="sign-up-submit" type="submit">Sign Up</button>
+            </form>
+        </div>
+
+        {/* This is the login form */}
+        <div className="form-container sign-in-container">
+            <form onSubmit={handleLoginSubmit}>
+            <h1>Login</h1>
+            <input id="login-email" type="email" placeholder="Email" value={ email } onChange={handleEmailChange} />
+            <input id="login-password" type="password" placeholder="Password" value={ password } onChange={handlePasswordChange} />
+            {errorMessage && (
+            <div className="error-message">
+
+                <p>{errorMessage}</p>
+            </div>
+            )}
+
+            <button id="login-submit" type="submit">Submit</button>
```

Comment:
Could this button say something like 'Log in' to match the syntax on the sign up button? Marie will like this. Submit is very standard practice but also pretty meaningless in accessibility terms.